### PR TITLE
330 integrate manatal job portal embed

### LINKED ISSUE #330: Integrate Manatal Job portal embed
AC:
HR / Talent Management need to migrate https://www.netcentric.biz/jobs from Smartrecruiters to Manatal #manatal-jira-integration-extravaganza 
The Manatal API need to be embeded so we can display the job offers as it is done today from Smartrecuiters platform

### PR DESCRIPTION
**Pull request by biljana-popovic**
Fix #330 

Test URLs:
- Before: https://main--cn-website--netcentric.hlx.page/jobs
- After: https://330-integrate-manatal-job-portal-embed--cn-website--netcentric.hlx.page/jobs

### CODE REVIEW

File: blocks/job-openings/job-openings.js [hunk 1]
```diff
@@ -1,190 +1,11 @@
...
+const jobOpeningsBlockItem = jobOpeningsBlock.item(0);
+const emptyDiv = jobOpeningsBlockItem.firstElementChild;
 
-let jobOpenings;
-let jobListOffset = 0;
-let selectedPosition = 'all';
-let selectedLocation = 'all';
+range.selectNode(jobOpeningsBlockItem);
+const documentFragment = range.createContextualFragment(tagString);
+jobOpeningsBlockItem.appendChild(documentFragment);
```
Comment (nc-andreashaller) [inline blocks/job-openings/job-openings.js:9]: typically we have used `innerHTML` or the DOM API in other places for adding HTML, any reason to do it via fragment here?

File: .env
Comment (nc-andreashaller) [inline .env:1]: for what is this needed?

Comment (biljana-popovic) [reply .env:1]: This is something I have from the moment I set up the project, from the guidelines when I was in onboarding process.

File: blocks/job-openings/job-openings.js (hunk 1 above)
Comment (biljana-popovic) [reply blocks/job-openings/job-openings.js:9]: After the discussion with @jonatan-lledo-netcentric , we went with this approach instead innerHTML.

File: blocks/job-openings/job-openings.js [hunk 2]
```diff
@@ -1,190 +1,11 @@
-import { readBlockConfig } from '../../scripts/lib-franklin.js';
+const tagString = '<iframe src="//www.careers-page.com/netcentric" class="embed-job-openings"></iframe>';
+const range = document.createRange();
+const jobOpeningsBlock = document.querySelectorAll('div.job-openings');
```
Comment (jonatan-lledo-netcentric) [inline blocks/job-openings/job-openings.js:3]: why querySelectorAll, and also, why not use the decorate function? that fuction already provides the block element to start with

File: blocks/job-openings/job-openings.js (hunk 1 above)
Comment (jonatan-lledo-netcentric) [reply blocks/job-openings/job-openings.js:9]: yes I recently learn about the fragment function and will be safer that use the innerHTML IMHO

Comment (nc-andreashaller) [reply blocks/job-openings/job-openings.js:9]: curious, what is safer about it?

Comment (jonatan-lledo-netcentric) [reply blocks/job-openings/job-openings.js:9]: innerHTML is used to inject malicious code, so if there is no innerHTML, then there are also no injections at all. That's why

File: .env
Comment (nc-andreashaller) [reply .env:1]: I think it is not needed anymore and can be removed.

File: blocks/job-openings/job-openings.js (hunk 1 above)
Comment (nc-andreashaller) [reply blocks/job-openings/job-openings.js:9]: but how does this prevent XSS? isn't HTML interpreted in the same way?

**Review by nc-andreashaller — APPROVED** [review]

**Comment by alexnerdrumhansen** [comment]
Things mostly seem to work.
A couple of things to note:

- Background image “Cognizant Netcentric” too large and misaligned
- 
Screenshot 2023-06-12 at 15 48 10

- “x departments selected” has the last word showing outside the drop down box
-
Screenshot 2023-06-12 at 15 34 46

- “before_js” appears at the top of the job list
- 
Screenshot 2023-06-12 at 15 45 34

**Comment by tomasznetcentric** [comment]
@alexnerdrumhansen Yes, I'm agree. This looks terrible but it is content of the Iframe. I'm not sure if I have influence on this.

**Pull request merged**